frontend/src/v5/ui/routes/viewer/drawings/drawingsList/drawingsList.component.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { DrawingsHooksSelectors } from '@/v5/services/selectorsHooks';
+import { DrawingItem } from './drawingItem/drawingItem.component';
+import { List } from './drawingsList.styles';
+import { CentredContainer } from '@controls/centredContainer';
+import { Loader } from '@/v4/routes/components/loader/loader.component';
+
+export const DrawingsList = () => {
+	const drawings = DrawingsHooksSelectors.selectCalibratedDrawings();
+	const isLoading = DrawingsHooksSelectors.selectAreStatsPending();
+
+	if (isLoading) return (
+		<CentredContainer>
+			<Loader />
+		</CentredContainer>
+	);
+
+	return (
+		<List>
```

Comment:
The mockup doesn't include this inner container. Or is this being left for a later PR which will remove it from the tickets list as well?

Reply:
I'm just trying to stick to the design we currently use for other cards, as mockups are often different. @sanmont3drepo can you confirm whether I'm doing the right thing here?

Reply:
Yes if we make it so they use the same design so if we change one we  change all listing styles - that would be ideal, even if it deviates from the design.